tests/phpunit/Admin/WC_Stripe_REST_OC_Setting_Toggle_Controller_Test.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace WooCommerce\Stripe\Tests\Admin;
+
+use Exception;
+use WC_Stripe_REST_OC_Setting_Toggle_Controller;
+use WC_Stripe_UPE_Payment_Gateway;
+use WooCommerce\Stripe\Tests\WC_Mock_Stripe_API_Unit_Test_Case;
+use Automattic\WooCommerce\Blocks\Package;
+use WP_REST_Request;
+
+/**
+ * Class WC_Stripe_REST_OC_Setting_Toggle_Controller_Test
+ *
+ * WC_Stripe_REST_OC_Setting_Toggle_Controller unit tests.
+ */
+class WC_Stripe_REST_OC_Setting_Toggle_Controller_Test extends WC_Mock_Stripe_API_Unit_Test_Case {
+	/**
+	 * Tested REST route.
+	 */
+	const SETTINGS_ROUTE = '/wc/v3/wc_stripe/oc_setting_toggle';
+
+	/**
+	 * Controller instance
+	 *
+	 * @var WC_Stripe_REST_OC_Setting_Toggle_Controller
+	 */
+	private $controller;
+
+	/**
+	 * Enable UPE and store gateway instance.
+	 *
+	 * We are doing this here because if we did it in set_up(), the method body would get called before every single test
+	 * however the REST controller is instantiated only once. If we reloaded gateways then, WC()->payment_gateways()
+	 * would contain another gateway instance than the controller.
+	 *
+	 * @see UPE_Test_Utils::reload_payment_gateways()
+	 */
+	public static function set_up_before_class() {
+		parent::set_up_before_class();
+
+		require_once WC_STRIPE_PLUGIN_PATH . '/includes/admin/class-wc-stripe-rest-oc-setting-toggle-controller.php';
+
+		// All tests assume UPE is enabled.
+		update_option( '_wcstripe_feature_upe', 'yes' );
```

Comment:
Nit: we may not need to enable UPE anymore as UPE is the default now.

Reply:
You're right! Removed it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4495/commits/1f748af33ae2539a55a60159720fb825a444b5e6